packages/webapp/src/inductions/interfaces.ts

```diff
@@ -16,12 +16,23 @@ export interface Induction {
     new_member_profile: NewMemberProfile;
 }
 
+export enum InductionRole {
+    Inviter = "inviter",
+    Endorser = "endorser",
+    Invitee = "invitee",
+    Member = "member", // use?
+    NonMember = "non-member", // use?
+    Unauthenticated = "unauthenticated",
+    Unknown = "unknown",
```

Comment:
out of curiosity, any reason why we need named enums? if it's not being used we could leave it as a plain enum....

Reply:
I'll double check where/how we're using those enums. But plain enum is prob fine.

Reply:
Yeah I noticed that we needed named enums in the genesis steps comparisons since we can't have `instanceof` enums. But here I don't think so... so cleaning it up is just easier to maintain